groups/font-features.yml

```
@@ -0,0 +1,3 @@
+# https://drafts.csswg.org/css-fonts-4/#font-rend-props
+name: Font features
```

And in the spirit of the previous comment, would this be out of line?

```suggestion
name: Typographic font features
```

---

I think that makes sense- should I also change the group key to `typographic-font-features`?